src/reports/ChartReportPage.jsx

```diff
@@ -20,6 +20,9 @@ import {
 import useReportStyles from './common/useReportStyles';
 
 const ChartReportPage = () => {
+  const [, setZoomDomain] = useState({ start: 'dataMin', end: 'dataMax' });
+  const handleBrushChange = (range) => { if (range) { setZoomDomain({ start: range.startIndex, end: range.endIndex }); } };
```

Comment:
What is the point of this?

Reply:
Hello @tananaev, good day to you, so sorry was a bit busy, that is how i manage the zoom state. 
`const [, setZoomDomain]` holds the zoom range and  
`const handleBrushChange` updates it based on user interaction.
The first value of `const [, setZoomDomain]` zoomDomain gives error on lint that it's declared and not used, i figure the function could work without it.

Reply:
If you don't use the value anywhere, I don't understand how it works.

Reply:
Yes Anton, it is not used anywhere in the code, it is only to hold and set the initial values for the `setZoomDomain` function, these initial values are provided by `dataMin` and `dataMax` , if you don't mind i could put it back but it will fail linting.

Reply:
Not sure I understand the explanation. Why set some values if they are not used anywhere?

Reply:
The `useState` is used to keep track of the zoom level of the chart and change when user interact with the `brush.`
Initially, `zoomDomain` starts with `{ start: 'dataMin', end: 'dataMax' }`. This means the whole data set is visible when the chart is first loaded.
`handleBrushChange` is triggered when users interact with `brush`, `setZoomDomain` updates the `zoomDomain` state with the new start and end points, effectively zooming into the selected range.
`handleBrushChange(range)` returns the value stored in `zoomDomain` so it is actually it's values that are in use.
i'm sorry i really don't know how to explain better for you.

https://legacy.reactjs.org/docs/hooks-state.html
the `zoomDomain` could be any name i decided to use nothing because it was failing linting

Reply:
Essentially updating the `zoomDomain` state in the `handleBrushChange` function, but nothing in the component is actually using the value of `zoomDomain`

Reply:
Let me ask in a different way. What if we completely remove these two lines and `onChange` parameter. What will change in the behavior?

Reply:
Anton i removed the two const they don't seem to be necessary.